### ISSUE TITLE
tensorflow can't support float64 type when reading from csv data?

### ISSUE THREAD

**Issue by liumilan** [tensorflow/tensorflow#2260]
I just follow the  examples,and tried to read csv data with float types

filename_queue = tf.train.string_input_producer(["test1.csv"])

reader = tf.TextLineReader()
key, value = reader.read(filename_queue)

record_defaults = [[1], [1.0], [1.0], [1.0], [1.0]]
col1,col2,col3,col4, col5 = tf.decode_csv(
    value, record_defaults=record_defaults)
features = tf.pack([col1, col2, col3, col4])

with tf.Session() as sess:
  #print sess.run(value)
  # Start populating the filename queue.
  coord = tf.train.Coordinator()
  threads = tf.train.start_queue_runners(coord=coord)
  print key,value
  for i in range(1200):
    # Retrieve a single instance:
    example, label = sess.run([features, col5])

  coord.request_stop()
  coord.join(threads)
~ 
But it need to read data into memory first.Because the csv file it too big to read into memory.Can it possbile read with shuffle and minibatch and then training?

**Comment by petewarden** [comment]
Sorry you're hitting problems! I'm a bit confused though - the title indicates a problem reading float64 values, but the text talks about hitting memory usage problems.

Do you have an example of the error message you're hitting? That might help.

**Comment by terrytangyuan** [comment]
You can checkout some examples in skflow using dask if it doesn't fit in memory.  You can also use pandas to load csv files.

**Comment by mrry** [comment]
Closing this issue due to inactivity. If you revisit it, @liumilan, I think this might be a better question for Stack Overflow....
